item.rb

```diff
@@ -0,0 +1,25 @@
+require 'date'
+
```

Comment:
Great job team 🤝

- [ ] As stated in the project requirements, kindly ensure all associations (1-to-many relationships) are implemented. Kindly make sure the `Genre`, `Author`, and `Label` Classes have many items implemented. 
- Use this as an example of Genre to guide you.
![genre](https://github.com/codedit334/Ruby_Capstone_Project/assets/63917725/d47e1bf7-ea4e-4898-9ae8-12d0c832ad24)

Reply:
Well Resolved ✔️ 

![image](https://github.com/codedit334/Ruby_Capstone_Project/assets/38088818/335cd0aa-cf8b-4de0-ba86-148d1a744343)